src/domTree.js

```diff
@@ -61,20 +63,21 @@ class span implements CombinableDomNode {
     depth: number;
     width: ?number;
     maxFontSize: number;
-    style: {[string]: string};
+    style: CssStyle;
     attributes: {[string]: string};
 
     constructor(
         classes?: string[],
         children?: DomChildNode[],
         options?: Options,
+        style?: CssStyle,
     ) {
         this.classes = classes || [];
         this.children = children || [];
         this.height = 0;
         this.depth = 0;
         this.maxFontSize = 0;
-        this.style = {};
+        this.style = Object.assign({}, style);
```

Comment:
I double checked the build output and the polyfill for this is included.  I think this will be smaller than object spread, but not sure.

Reply:
TIL: that the second argument to Object.assign() can be null or undefined. This is great!